docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+
+### Simulation conditions
+Sometimes, several different experiments are performed on a system (each potentially generating several measurements). An experiment could e.g. be the time development of a system from a specific initial condition. Since each experimental condition (during the optimisation procedure, for a guess of the unknown parameters) generates a distinct simulation, these are also called simulation conditions. In our example, all data comes from a single experiment, and the simulation condition input is not required. How to define and use different experimental conditions is described [here](@ref petab_simulation_conditions).
+
+
+### Measurements
+Finally, we need to define the system measurements to which the parameters will be fitted. Each measurement combines:
+1. The observable which is observed (here we use the id tag defined in the `observables` dictionary).
+2. The time point of the measurement.
+3. The measured value.
+  
+For cases where several simulation conditions are given, we also need to provide:
+
+4. The simulation condition which generates the measurement ([here](@ref petab_simulation_conditions) is an example where this is used).
+
+([When [pre-equilibration](https://sebapersson.github.io/PEtab.jl/stable/Brannmark/) is used to initiate the system in a steady state, a fifth field is also required])
+
+Measurements are provided in a `DataFrame` where each row corresponds to a measurement. The values are provided in the `obs_id`, `time`, `measurement`, and `simulation_id` columns. In our case we only need to fill in the first three:
+```petab1
+using DataFrames
+measurements = DataFrame(obs_id="obs_P", time=sol.t[10:10:end], measurement=data)
+```
+Since, in our example, all measurements are of the same observable, we can set `obs_id="obs_P"`. If several different observables were measured, `obs_id` would be a vector of the same length as `time` and `measurement`.
+
+### Creating a PEtabModel
+Finally, we combine all inputs in a single `PEtabModel`. To it, we also pass the initial conditions of our simulations (using the `state_map` argument). It is also possible to have [initial conditions with uncertainty](@ref petab_simulation_initial_conditions_uncertainty), [that vary between different simulations](@ref petab_simulation_conditions), or [that we attempt to fit to the data](@ref petab_simulation_initial_conditions_fitted).
+```petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+
+### Fitting parameters
+We are now able to fit our model to the data. First, we create a `PEtabODEProblem`. Here, we use `petab_model` as the only input, but it is also possible to set various [numeric solver and automatic differentiation option](@ref petab_simulation_conditions) (such as method or tolerance).
```

Comment:
Mention briefly that if nothing is chosen PEtab.jl typically chooses good default options based on the problem

Reply:
updated